CSS/Comments.tmPreferences

```diff
@@ -14,13 +14,13 @@
 				<key>name</key>
 				<string>TM_COMMENT_START</string>
 				<key>value</key>
-				<string>/*</string>
+				<string>/* </string>
```

Comment:
This seems inconsistent with the other languages in the default packages, and is more of a personal preference. Since there isn't really a functional difference, I'd prefer to defer to it staying the same. Mostly because users are particular about "unnecessary" changes.

We discussed this some in #228.